pytensor/link/pytorch/dispatch/elemwise.py

```diff
@@ -37,6 +38,64 @@ def dimshuffle(x):
     return dimshuffle
 
 
+@pytorch_funcify.register(Sum)
+def pytorch_funcify_sum(op, **kwargs):
+    def torch_sum(x):
```

Comment:
Nitpick, any reason why here you do op.axis, and in the other implementations you explicitly extract the axis? If this works fine, we can save lines in the other implementations?

Reply:
Sounds good. Also I noticed the Ops also have a `keepdims` argument.
But `op.keepdims` does not extract this

Reply:
keepdims is actually not a property of the Op but a kwarg to the helper function. It just introduces an expand_dims (dimshuffle) on the output of CAReduce, so we don't have to do anything